crypto3/libs/multiprecision/test/CMakeLists.txt

```diff
@@ -57,7 +57,7 @@ macro(define_big_int_test name)
     # target_link_libraries(${test_name} no_eh_support)
     add_dependencies(${CURRENT_PROJECT_NAME}_test_suite_big_int_tests ${test_name})
 
-endmacro(define_big_uint_test)
```

Comment:
Are we sure about this?
Now it complains:
```
CMake Warning (dev) in crypto3/libs/multiprecision/test/CMakeLists.txt:
  A logical block opening on the line

    /home/olekhov/work/placeholder/crypto3/libs/multiprecision/test/CMakeLists.txt:52 (macro)

  closes on the line

    /home/olekhov/work/placeholder/crypto3/libs/multiprecision/test/CMakeLists.txt:60 (endmacro)

  with mis-matching arguments.
```

Reply:
I think it was complaining before this fix